x-pack/plugins/secops/public/components/links/index.tsx

```diff
@@ -34,6 +34,44 @@ export const GoogleLink = pure<{ children?: React.ReactNode; link: string }>(
   )
 );
 
+export const PortOrServiceNameLink = pure<{
+  children?: React.ReactNode;
+  portOrServiceName: number | string;
+}>(({ children, portOrServiceName }) => (
+  <EuiLink
+    data-test-subj="port-or-service-name-link"
+    href={`https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=${portOrServiceName}`}
```

Comment:
I would add a `encodeURI(portOrServiceName)` here to be consistent and add a bit of safety in case someone added a port with a space on it.